### ISSUE TITLE
Broken link to documentation in Server Preparations of readme

### ISSUE THREAD

**Issue by isc-pbarton** [intersystems-community/intersystems-testingmanager#31]
This link in the README is broken:
```
If it doesn't exist, follow [these instructions](https://intersystems-community.github.io/vscode-objectscript/serverside/#configuring-storage-for-folder-specific-settings) to create it.
```

I think the new link should be https://docs.intersystems.com/components/csp/docbook/DocBook.UI.Page.cls?KEY=GVSCO_ssworkflow#GVSCO_ssworkflow_config_folderspec

**Comment by gjsjohnmurray** [comment]
@isc-pbarton thanks for the report. I have fixed this for the next release and added information about using an IPM package to set up the webapp.

**Comment by gjsjohnmurray** [comment]
Fix is in 0.23 which has just been published.
